src/js/letters/reducers/index.js

```diff
@@ -104,6 +108,13 @@ function letters(state = initialState, action) {
       return _.set(['letterDownloadStatus', action.data], 'failure', state);
     case UPDATE_ADDRESS:
       return _.set('address', action.address, state);
+    case SAVE_ADDRESS_PENDING:
+      return _.set('savePending', true, state);
+    case SAVE_ADDRESS_SUCCESS: {
+      const newState = Object.assign({}, state, { savePending: false });
+      return _.set('address', action.address, newState);
+    }
+    // Add SAVE_ADDRESS_FAILURE
```

Comment:
I don't know what's supposed to happen when we get a failure, so this is still on the TODO list, but I'm tracking it [here](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/4858).